src/main/venmic.ts

```diff
@@ -51,17 +54,35 @@ ipcMain.handle(IpcEvents.VIRT_MIC_LIST, () => {
         : { ok: false, isGlibcxxToOld };
 });
 
-ipcMain.handle(IpcEvents.VIRT_MIC_START, (_, targets: string[]) =>
-    obtainVenmic()?.link({
+ipcMain.handle(IpcEvents.VIRT_MIC_START, (_, targets: string[], workaround?: boolean) => {
+    const data: LinkData = {
         include: targets.map(target => ({ key: "application.name", value: target })),
         exclude: [{ key: "application.process.id", value: getRendererAudioServicePid() }]
-    })
-);
+    };
+
+    if (workaround) {
+        data.workaround = [
+            { key: "application.process.id", value: getRendererAudioServicePid() },
```

Comment:
nit: it would be better to store the result of this in a variable to avoid the extra work (i assume collecting metrics is kinda costly)

Reply:
True, good idea

Reply:
Can we use a global cached variable?